packages/block-editor/src/hooks/test/use-editor-wrapper-styles.native.js

```diff
@@ -0,0 +1,475 @@
+/**
+ * External dependencies
+ */
+import { renderHook } from '@testing-library/react-native';
+
+/**
+ * WordPress dependencies
+ */
+import { getBlockTypes, unregisterBlockType } from '@wordpress/blocks';
+import { registerCoreBlocks } from '@wordpress/block-library';
+
+/**
+ * Internal dependencies
+ */
+import {
+	getBlockMargin,
+	getFullWidthStyles,
+	getWideWidthStyles,
+	useEditorWrapperStyles,
+} from '../use-editor-wrapper-styles';
+
+jest.mock( '../use-editor-wrapper-styles.scss', () => ( {
```

Comment:
Exactly, because our current [setup overrides](https://github.com/WordPress/gutenberg/blob/trunk/test/native/jest.config.js#L47) all `.scss` files with the `styleMock` file. Maybe in the future, we should keep mocking styles within the test itself.